locales/th/common.json

```diff
@@ -1,3 +1,5 @@
 {
-  "greeting": "สวัสดี"
+  "greeting": "สวัสดี",
```

Comment:
Same as this [comment](https://github.com/Cobda/cobda-web/pull/25/files#r579642002)